decisions/2019/05/25-war-docker-schemes.md

```diff
@@ -0,0 +1,92 @@
+# Name scheme for `WAR`-files and Docker images
+
+## `WAR`-file
+
+#### Stable
+
+URL structure: `/{version}.war`
+                 
+- https://releases.dhis2.org/2.32.0.war
+- https://releases.dhis2.org/2.31.4.war
+
+#### Canary
+
+URL structure: `/canary/{branch}(-{date}).war`
+
+- https://releases.dhis2.org/canary/master.war
+- https://releases.dhis2.org/canary/2.32.war
+- https://releases.dhis2.org/canary/2.32-20190523.war
+
+Date is optional.
+
+#### Development
+
+URL structure: `/dev/{branch}.war`
+
+- https://releases.dhis2.org/dev/2.31.war
+- https://releases.dhis2.org/dev/2.32.war
+- https://releases.dhis2.org/dev/master.war
+
+#### Pull requests
+
+URL structure: `/pr/{pull-id}.war`
+
+- https://releases.dhis2.org/pr/3353.war
+- https://releases.dhis2.org/pr/239.war
+
+## Docker
+
+### Base images
+
+#### Stable
+
+Repo: `core`
+Tag structure: `{version}(-{variant})`
+
+- `dhis2/core:latest`
+- `dhis2/core:2.32.0`
+- `dhis2/core:2.31.3`
+
+#### Canary
+
+Repo: `core-canary`
+Tag structure: `{branch}(-{date})(-{variant})`
+
+- `dhis2/core-canary:master`
+- `dhis2/core-canary:2.32`
+- `dhis2/core-canary:2.32-20190522`
+- `dhis2/core-canary:2.31-20190522`
+- `dhis2/core-canary:master-20190522`
+
+#### Development
+
+Repo: `core-dev`
+Tag structure: `{branch}(-{variant})`
+
+- `dhis2/core-dev:master`
+- `dhis2/core-dev:2.32`
+- `dhis2/core-dev:2.31`
+
+#### Pull requests
+
+Repo: `core-pr`
+Tag structure: `{pull-id}-{trimmed branch name}`
```

Comment:
Why couldn't this just be the PR ID?  Does adding the trimmed branch name help us somehow?

Reply:
It gives the reader some indication of what the PR is about. Though it's going to be an exercise in copy-paste from Docker Hub so PR ID will be a lot easier to predict.

I'll change.

Reply:
406fe00